packages/core/test/interpreter.test.ts

```diff
@@ -1483,16 +1409,16 @@ describe('interpreter', () => {
       expect(() => {
         const state$ = from(intervalService);
 
-        state$.subscribe(
-          () => {
+        state$.subscribe({
+          next: () => {
```

Comment:
the former version is still allowed in the Interpreter - so I wonder, why changing this here?